src/Operators.jl

```diff
@@ -37,26 +37,41 @@ end
 function cube(x::T)::T where {T<:Real}
     return x^3
 end
-function pow_abs(x::T, y::T)::T where {T<:Real}
-    return abs(x)^y
+function safe_pow(x::T, y::T)::T where {T<:Real}
+    if isinteger(y)
```

Comment:
Would it be faster if this was `T <: Integer`?

Also - I'm not sure if `T(NaN)` will work if it's an integer.

Reply:
Actually I think there's just not any good way to have safe operations for integers. I think this piece of the code could be removed for the time being. We can come back to such types later, but not needed for this PR.

Reply:
Wait, I misunderstood what `isinteger` actually does. This looks good then!